bin/next-build-tools.js

```diff
@@ -136,6 +137,20 @@ program
 		});
 
 	program
+		.command('deploy-vcl [folder]')
```

Comment:
can we make this `deploy-vcl [file]` would be a bit simpler since this only support a single file anyway